department-of-veterans-affairs/abd-vro
# MCP 2058: Add recordedDate to condition

## What was the problem?

PDF design requires conditions to be sorted by recordedDate, not onsetDate.

Additional context from Carolyn, 
“““

It’s because the onset date column is usually inaccurate, so the only reliable data is  recorded date.

Background detail: We noticed when looking at production data that the onset date” and recorded date were usually exactly the same. We asked why, and the Lighthouse team explained that when there was no data in onset date, they insert recorded date

Associated tickets or Slack threads:

- [MCP-2058](https://amida.atlassian.net/browse/MCP-2058)

## How does this fix it?[^1]


## How to test this PR
- Step 1 Build and run project. Try health-assessment endpoint for diagnosticCode "0" an verify that recordedDate is populated. 
- Step 2[^secrel]


[^1]: [Pull-Requests guidelines](https://github.com/department-of-veterans-affairs/abd-vro/wiki/Pull-Requests). If PR is significant, update [Current Software State](https://github.com/department-of-veterans-affairs/abd-vro/wiki/Current-Software-State) wiki page.
[^secrel]: To check if a PR will succeed in the SecRel workflow, [test PRs in the SecRel pipeline](https://github.com/department-of-veterans-affairs/abd-vro-internal/wiki/Secure-Release-process#to-test-prs-in-the-secrel-pipeline).
